src/main/resources/jp/vmi/html/result/result.html

```diff
@@ -1,396 +1,464 @@
+            </tr>
+            ${end}
+            ${if selenese.type="TEST_CASE"}
+            <tr class="status_${selenese.result}">
+                <td><a href="#testCase${index_selenese}">${selenese.name;h}</a></td>
+            </tr>
+            ${end}
+            ${end}
         </table>
-      </div>
     </div>
+</div>
 
 ${foreach seleneseList selenese}
-  ${if selenese.type="TEST_CASE"}
-    <br>
-    <div class="tabs" id="tab${index_selenese}">
-      <ul>
+${if selenese.type="TEST_CASE"}
+<br>
+<div class="tabs" id="tab${index_selenese}">
+    <ul>
         <li>
-          <input name="tab${index_selenese}" id="tab${index_selenese}_source" type="radio" checked>
-          <label for="tab${index_selenese}_source" class="tab_source" onclick="selectTab(this)"><a name="testCase${index_selenese}">Source</a></label>
-          <div class="contents">
-            <table class="info"><tr><th>Source:</th><td>${selenese.filename;h}</td></tr></table>
-            <table cellpadding="1" cellspacing="1" border="1">
-              <thead>
-                <tr class="title status_${selenese.result}"><td class="indexh">#</td><td rowspan="1" colspan="3">${selenese.name;h}</td><td>&Delta;t(ms)</td><td>Screenshot</td></tr>
-              </thead>
-              <tbody>
-    ${foreach selenese.commandList command}
-                <tr class="cmd_status_${command.result}">
-                  <td class="index">${command.index;i}</td>
-      ${foreach command.source item}
-                  <td>${item;h}</td>
-      ${end}
-                  <td class="number">${@commandResults results selenese command}${foreach results result , }${result.duration;n}${end}</td>
-                  <td>${foreach command.screenshots screenshot , }<a href="${screenshot.path;relpath}">${index_screenshot}:${screenshot.label;h}</a>${end}</td>
-                </tr>
-    ${end}
-              </tbody>
-            </table>
-          </div><!-- contents -->
+            <input name="tab${index_selenese}" id="tab${index_selenese}_source" type="radio" checked>
+            <label for="tab${index_selenese}_source" class="tab_source" onclick="selectTab(this)"><a name="testCase${index_selenese}">Source</a></label>
+            <div class="contents">
+                <table class="info">
+                    <tr>
+                        <th>Source:</th>
+                        <td>${selenese.filename;h}</td>
+                    </tr>
+                </table>
+                <table cellpadding="1" cellspacing="1" border="1">
+                    <thead>
+                    <tr class="title status_${selenese.result}">
+                        <td class="indexh">#</td>
+                        <td rowspan="1" colspan="3">${selenese.name;h}</td>
+                        <td>&Delta;t(ms)</td>
+                        <td>Screenshot</td>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    ${foreach selenese.commandList command}
```

Comment:
it seems like this gets parsed although it is below this ${if selenese.type="TEST_CASE"} for Testsuite s2 in jp.vmi.html.result.HtmlResultTest